samples/build/build_buildpacks-v3_cr.yaml

```
@@ -1,16 +1,17 @@
 ---
-apiVersion: shipwright.io/v1alpha1
+apiVersion: shipwright.io/v1beta1
 kind: Build
 metadata:
   name: buildpack-nodejs-build
-  annotations:
-    build.shipwright.io/build-run-deletion: "false"
 spec:
   source:
-    url: https://github.com/shipwright-io/sample-nodejs
+    git: 
+      url: https://github.com/shipwright-io/sample-nodejs
     contextDir: source-build
   strategy:
     name: buildpacks-v3
     kind: ClusterBuildStrategy
+  retention:
+    atBuildDeletion: true
```

Because `build-run-deletion` annotation was `false` here.

```suggestion
  retention:
    atBuildDeletion: false
```